### PR TITLE
Fix webpack browser install

### PR DESCRIPTION
**Pull request by stevemk14ebr**
Fixes https://github.com/handlebars-lang/handlebars.js/issues/1174

In webpack 5 things seem to have changed. From https://github.com/webpack/webpack/issues/15007 which itself is from https://github.com/facebook/create-react-app/issues/11756 the fs polyfills webpack used to include by default are no longer included. The browser wildcard '.' isn't valid according to https://github.com/webpack/webpack/issues/15007 and should refer to a full filepath. I've changed the config to be valid, and it seems to work as far as I can tell. There was also a missing coffeescript dev dependency which caused the grunt build to fail, I added this too.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,12 +70,13 @@
     "uglify-js": "^3.1.4",
     "underscore": "^1.5.1",
     "webpack": "^1.12.6",
-    "webpack-dev-server": "^1.12.1"
+    "webpack-dev-server": "^1.12.1",
+    "coffeescript": "^2.6.1",
```
Comment (jaylinski) [inline package.json:74]: Why did you add `coffeescript` as a dependency?

**Review by jaylinski — CHANGES_REQUESTED** [review]
This should be backported to the `4.x` branch.

File: package.json (hunk 1 above)
Comment (stevemk14ebr) [reply package.json]:
```
 ~/source/repos/tmp/aster/node_modules/handlebars  npm run build

> handlebars@5.0.0-alpha.1 build
> grunt build

Loading "bg-shell.coffee" tasks...ERROR
>> Error: Grunt attempted to load a .coffee file but CoffeeScript was not installed.
>> Please run `npm install --dev coffeescript` to enable loading CoffeeScript.
Loading "child-process.coffee" tasks...ERROR
>> Error: Grunt attempted to load a .coffee file but CoffeeScript was not installed.
>> Please run `npm install --dev coffeescript` to enable loading CoffeeScript.

Running "babel:cjs" (babel) task

Running "webpack:handlebars" (webpack) task
Version: webpack 1.15.0
        Asset    Size  Chunks             Chunk Names
handlebars.js  572 kB       0  [emitted]  main

Running "webpack:runtime" (webpack) task
Version: webpack 1.15.0
                Asset    Size  Chunks             Chunk Names
handlebars.runtime.js  383 kB       0  [emitted]  main

Done.
```

**Comment by stevemk14ebr** [comment]
the big diff from that is just from josn formatting ^

**Comment by chrisl777** [comment]
@stevemk14ebr @jaylinski Any update on this PR? I am hoping to be able to use handlebars with create-react-app (webpack 5 under the hood, presumably).

**Comment by jaylinski** [comment]
I'll have a look at it this weekend.

**Comment by jaylinski** [comment]
I added a test for this bug and opened a new PR: https://github.com/handlebars-lang/handlebars.js/pull/1862

I'll merge and tag it (hopefully) next week.

**Comment by chrisl777** [comment]
@jaylinski Awesome, thank you!

**Comment by stevemk14ebr** [comment]
Bump on doing a release on npm?

**Comment by jaylinski** [comment]
I released a new patch version (v4.7.8) von npm. Sorry it took so long.
